src/main/java/commands/EventCommand.java

```diff
@@ -0,0 +1,62 @@
+package commands;
+
+import java.time.DateTimeException;
+
+import applemazer.Storage;
+import applemazer.TaskList;
+import tasks.Event;
+import tasks.Task;
+
+/**
+ * Class that represents the "event" command.
+ */
+public class EventCommand extends Command {
+    private final String desc;
+    private final String from;
+    private final String to;
+
+    /**
+     * Constructor for an {@code EventCommand} object.
+     * <p>
+     * Note that setting the time is optional.
+     * @param desc Description of the event.
+     * @param from The start date (and time) of the event, as a String.
+     * @param to The end date (and time) of the event, as a String.
+     */
+    public EventCommand(String desc, String from, String to) {
+        this.desc = desc;
+        this.from = from;
+        this.to = to;
+    }
+
+    /**
+     * Executes the "event" command which adds an {@code Event} task to the task list.
+     * @param tasks   The task list to use.
+     * @param storage The storage object containing the filepath which the chatbot saves to and loads from.
+     */
+    @Override
+    public void execute(TaskList tasks, Storage storage) {
+        try {
+            Task task = new Event(desc, from, to);
+            tasks.add(task);
+            task.printTaskAddedMessage(tasks.size());
+            storage.save();
+        } catch (DateTimeException e) {
```

Comment:
I noticed that the current implementation allows the creation of events with invalid dates, such as when the 'to' date is earlier than the 'from' date. For example:

``` 
event meeting /from 2022-12-12 /to 2021-12-12
Got it. I've added this task: 
    [E][ ] meeting (from: 12 Dec 2022 to: 12 Dec 2021) 
Now you have 5 tasks in the list. 

Saving data...
Save successful.
```

This could lead to confusion and potentially incorrect data being stored. I suggest adding validation to check that the 'to' date is not earlier than the 'from' date before allowing the task to be created.